python/cugraph/dask/centrality/katz_centrality.py

```diff
@@ -115,7 +115,8 @@ def katz_centrality(input_graph,
     Examples
     --------
     >>> import cugraph.dask as dcg
-    >>> Comms.initialize(p2p=True)
+    >>> ... Init a DASK Cluster
+    >>    see https://docs.rapids.ai/api/cugraph/stable/dask-cugraph.html
```

Comment:
minor: since these are typically copy-and-pasteable snippets, you could use a comment char here:
```
>>> import cugraph.dask as dcg
>>> # ... Init a DASK Cluster
>>> #   see https://docs.rapids.ai/api/cugraph/stable/dask-cugraph.html
>>> chunksize = dcg.get_chunksize(input_data_path)
```

Reply:
true.  but it is not executable code.  There used to be a line initializing Comm, but that is not fully what is needed for DASK.  I pulled that out so that there is one place to maintain what setting up dask looks like.  If someone cut&pastes the above block, I would prefer it break on a non-executable line